### PR TITLE
CASMPET-7286 Release notes for 1.6

### PR DESCRIPTION
**Pull request by rajeshranganathan-hpe**
# Description

Release notes for platform products in CSM 1.6.0.
# Checklist
- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by Srinivas-Anand-HPE — APPROVED** [review]

**Review by arka-pramanik-hpe — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -16,27 +16,49 @@
...
 
 | Platform Component           | Version        |
 |------------------------------|----------------|
+| `Kubernetes`                 | 1.24.17        |
+| `istio`                      | 1.19.10        |
+| `kiali`                      | 1.75.0         |
+| `Strimzi Kafka`              | 0.41.0         |
+| `Kyverno`                    | 1.10.7         |
 
 ### Security improvements
```
Comment (yarlagadda-srinivasa-rao) [inline RELEASE_NOTES.md:44]: CVE fixes and security improvements from other components must be added here.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -51,14 +73,21 @@ For a list of all features with an announced removal target, see [Removals](intr
...
+    * When installing CSM 1.6, `cray-shared-kafka-kafka-` pods in the services namespace fail to come up which results in `cfs-api` pods in CLBO state.
+    * A workaround is presented in [CFS API pods in CLBO](troubleshooting/known_issues/cfs-api_pods_in_CLBO_state.md).
+* `istio-proxy` containers fail with too many open files.
+    * This may happen when any pod with `istio injection` enabled is started.
+    * A workaround is presented in [Istio-Proxy failing with too many open files](troubleshooting/known_issues/Istio-Proxy_failing_with_too_many_open_files.md)
+* IUF does not run the next stage for an activity
+    * During CSM upgrade, IUF reports that multiple sessions are in progress for an activity.
+    * A workaround is presented in [IUF does not run the next stage for an activity](troubleshooting/known_issues/iuf_unable_to_run_next_stage.md)
 
 ### Security vulnerability exceptions in CSM 1.6
```
Comment (yarlagadda-srinivasa-rao) [inline RELEASE_NOTES.md:108]: Any reason why this is newly introduced in 1.6? Until we provide the fix or patch we will not share OR publish all known vulnerabilities in the software.

**Review by yarlagadda-srinivasa-rao — COMMENTED** [review]
You captured all the content that I shared correctly. I will approve after I see the complete security improvements from other component owners.

**Review by sravani-sanigepalli — APPROVED** [review]

**Review by HareshKumarkj — APPROVED** [review]

**Review by Pankhuri-Rajesh — APPROVED** [review]

**Review by aasha-hpe — COMMENTED** [review]
Looks good to me.

**Review by aasha-hpe — APPROVED** [review]

**Review by dborman-hpe — APPROVED** [review]
LGTM

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -16,27 +16,55 @@
 
 * **IMPORTANT** Systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
 
+### New software support
```
Comment (jld-dehn-hpe) [inline RELEASE_NOTES.md:24]: Can we add the following under line 13 for "miscellaneous functionality"?

* The [System Admin Toolkit (SAT)](glossary.md#system-admin-toolkit-sat) is fully included in CSM. There is no longer a separate SAT product stream to install. SAT 2.6 releases, which accompanied CSM 1.5, are the last releases of SAT as a separate product. For more information, see [SAT in CSM](/operations/system_admin_toolkit/about_sat/SAT_in_CSM.md).

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -16,27 +16,55 @@
...
 
 ### Security improvements
 
+* `Kyverno` is upgraded from 1.9.5 version to 1.10.7 version to address CVEs.
+
 ### Customer-requested enhancements
 
+* iSCSI based boot content projection for `rootfs` and `PE` images.
+
 ### Documentation enhancements
```
Comment (jld-dehn-hpe) [inline RELEASE_NOTES.md:52]: Can we also add the following under line 46 for "documentation enhancements"?

* The SAT documentation moved to be fully included within the [System Admin Toolkit (SAT) section](https://cray-hpe.github.io/docs-csm/en-16/operations/system_admin_toolkit/) of the [CSM Administration Guide](https://cray-hpe.github.io/docs-csm/en-16/operations/).

**Review by jld-dehn-hpe — CHANGES_REQUESTED** [review]
Adding mention of SAT becoming fully integrated with CSM in the 1.6 release. @haasken-hpe, can you confirm that these changes should be mentioned in the 1.6 release notes?

File: RELEASE_NOTES.md (hunk 4 above)
Comment (rajeshranganathan-hpe) [reply RELEASE_NOTES.md:52]: Done

File: RELEASE_NOTES.md (hunk 3 above)
Comment (rajeshranganathan-hpe) [reply RELEASE_NOTES.md:24]: Done

**Review by jld-dehn-hpe — APPROVED** [review]
Looks great, thank you!

**Review by rustydb — APPROVED** [review]

**Pull request merged**